KubeArmor/config/config.go

```diff
@@ -32,6 +32,9 @@ type KubearmorConfig struct {
 	CoverageTest bool // Enable/Disable Coverage Test
 }
 
+// PolicyDir policy dir path for host policies backup
+const PolicyDir string = "/opt/kubearmor/policies/"
```

Comment:
Q1: we know the systemd mode uses /opt/kubearmor, but is it okay to have a static path in the code?

Reply:
But @nam-jaehyun . In that case also do we expect to store the policies ? we will be storing policies only on kvmagent mode and systemd mode right ?

Reply:
according to this code, we will also store policies in k8s if hostPolicy is enabled.

Reply:
Understood @nam-jaehyun 

We can check if policy is not enabled and then backup the policies. That should be a solution.